fontc_crater/src/args.rs

```diff
@@ -54,6 +55,7 @@ pub(super) struct CiArgs {
 
 #[derive(Debug, PartialEq, clap::Args)]
 pub(super) struct ReportArgs {
+    #[arg(default_value = "./font_repos/results.json")]
```

Comment:
I think drop this bit and make it be explicit. The report stuff isn't used for CI anyway and we can probably get rid at some point.

Reply:
Done.